src/graphql/admin/mongoose_graphql_domain_admin_query.erl

```diff
@@ -13,9 +13,9 @@ execute(_Ctx, admin, <<"domainsByHostType">>, #{<<"hostType">> := HostType}) ->
     {ok, Domains2};
 execute(_Ctx, admin, <<"domainDetails">>, #{<<"domain">> := Domain}) ->
     case mongoose_domain_sql:select_domain(Domain) of
-        {ok, #{host_type := HostType, enabled := Enabled}} ->
+        {ok, #{host_type := HostType, status := Status}} ->
             {ok, #domain{host_type = HostType, domain = Domain,
-                         enabled = Enabled}};
```

Comment:
I think that it would be better to define an Enum in GraphQL for Status, because this way we are losing information, and one would think a domain is just disabled, and try to re-enable, and fail, etc.. it would seem like a bug.